Description: Fix nimscriptapi import path
 The file is installed at /usr/lib/nim/nimblepkg/nimscriptapi.nim
Author: Federico Ceratto <federico@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/dist/nimble/src/nimblepkg/nimscriptsupport.nim
+++ b/dist/nimble/src/nimblepkg/nimscriptsupport.nim
@@ -174,6 +174,8 @@
 
 proc findNimscriptApi(options: Options): string =
   ## Returns the directory containing ``nimscriptapi.nim``
+  # Use /usr/lib/nim/nimblepkg/nimscriptapi.nim
+  if true: return "/usr/lib/nim"
   var inPath = false
   # Try finding it in exe's path
   if fileExists(getAppDir() / "nimblepkg" / "nimscriptapi.nim"):
